Example/StripeConnectExample/StripeConnectExample/MainViewController.swift

```diff
@@ -63,7 +63,18 @@ class MainViewController: UITableViewController {
     }
 
     lazy var embeddedComponentManager: EmbeddedComponentManager = {
-        return .init(appearance: AppSettings.shared.appearanceInfo.appearance,
+        /*
+         NOTE:
+         We are using a custom URL scheme in this example, but Stripe recommends
+         against doing this in production code to prevent malicious apps from
+         reusing the same URL scheme to intercept sensitive user data.
+
+         Instead, Stripe recommends using associated domains and adding the
+         return url to your websites `apple-app-site-association` file:
+         https://developer.apple.com/documentation/xcode/supporting-associated-domains
+         */
```

Comment:
Wording here needs to be tweaked – it sounds like we disallow custom URL schemes in livemode

Context:
https://stripe.slack.com/archives/C02KW8G938W/p1729217697015449

Reply:
turns out we don't need this